connectivity/FEATURE_BLE/source/generic/GapImpl.h

```diff
@@ -59,10 +59,15 @@ class BLEInstanceBase;
 
 class Gap :
     public ble::PalConnectionMonitor,
-    public PalGapEventHandler,
-    public PrivateAddressController::EventHandler {
+    public PalGapEventHandler
+#if BLE_FEATURE_PRIVACY
+    , public PrivateAddressController::EventHandler
+#endif //BLE_FEATURE_PRIVACY
+    {
     friend PalConnectionMonitor;
+#if BLE_FEATURE_PRIVACY
```

Comment:
How is it related to privacy ? We inherit unconditionally from this class.

Reply:
thanks, this is in error